R/submit-model.R

```diff
@@ -85,6 +85,47 @@ submit_model.bbi_nonmem_model <- function(
   return(res)
 }
 
+#' @describeIn submit_model Takes a `bbi_nmboot_model` object.
+#' @export
+submit_model.bbi_nmboot_model <- function(
+    .mod,
+    .bbi_args = NULL,
+    .mode = getOption("bbr.bbi_exe_mode"),
```

Comment:
Probably should be `.mode = "sge"` or maybe a boot exe mode option?

Reply:
I would consider setting up an `"bbr.bbi_nmboot_exe_mode"` option at some point, but I don't think it's needed for now.

In terms of the default... I guess maybe `"sge"` is safest. If someone has this set to `"local"` and they submit models in batches of 100... I guess it's likely to freeze up their machine.

And if they _want_ to submit locally, they can set this in their call (and probably reset `batch_size` too).

@barrettk if you agree, can you change this default to `.mode = "sge"`?